### ISSUE TITLE
ctags misses python tags

### ISSUE THREAD

**Issue by yevhen-m** [universal-ctags/ctags#1072]
I use ctags to generate tag files, but some of the definitions are missing.
I've figured out that triple quotes make ctags ignore all the rest of definitions in a file.

```
result = func('''some string''')

class C:
     pass
```
Now `C` is ignored.

**Comment by masatake** [comment]
Do you use the latest source tree?

```
[yamato@x201]~% cat >/tmp/foo.py <<EOF  
result = func('''some string''')

class C:
     pass
EOF
[yamato@x201]~% u-ctags -o - /tmp/foo.py
C	/tmp/foo.py	/^class C:$/;"	c
result	/tmp/foo.py	/^result = func('''some string''')$/;"	v
```

**Comment by yevhen-m** [comment]
```
Universal Ctags 0.0.0(5ab7c69), Copyright (C) 2015 Universal Ctags Team
Universal Ctags is derived from Exuberant Ctags.
Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert
  Compiled: Jun 22 2016, 12:50:31
  URL: https://ctags.io/
  Optional compiled features: +wildcards, +regex, +option-directory, +coproc, +xpath
```
That's outdated version, right?

**Comment by dkearns** [comment]
Yes, the Python parser has recently been overhauled.

**Comment by masatake** [comment]
> Compiled: Jun 22 2016, 12:50:31

New python parser is merged in Aug 1st.
```
commit 90cd3f28c14934f8b031f1af293df442d07914fc
Merge: 1b078ac 354f80e
Author: Colomban Wendling <ban@herbesfolles.org>
Date:   Mon Aug 1 14:38:36 2016 +0200

    Merge pull request #1058 from b4n/python-rewrite-rebase
    
    Python: rewrite as a tokenizing parser and add many new features.
    
    Fixes #997
    Closes #1003.
```

**Comment by yevhen-m** [comment]
Thanks for help.
